engine/index.py

```diff
@@ -217,6 +214,9 @@ def run(self):
                 val = self.queue_in.get(True, 1)
             except Empty:
                 continue
+            # Not sure why I needed to check this:
+            if val[self.sensor] is None:
+                continue
```

Comment:
@JonnoFTW Kept failing without this. Probably just a data issue. Feel free to remove, but I could not run without it.